src/snowflake/connector/cache.py

```diff
@@ -101,11 +100,7 @@ def _getitem(
             self._hit(k)
         return v
 
-    def _setitem(
-        self,
-        k: K,
-        v: V,
-    ) -> None:
+    def _setitem(self, k: K, v: V, **kwargs: Any) -> None:
```

Comment:
Why did you add this?

```suggestion
    def _setitem(self, k: K, v: V) -> None:
```

Reply:
it's no longer needed as I will use the `_update` method